services/121-service/src/registration/registrations.service.ts

```diff
@@ -817,42 +813,17 @@ export class RegistrationsService {
     // });
 
     // Get registration by referenceId
-    const registration = await this.getRegistrationOrThrow({
-      referenceId,
-      relations: ['programFinancialServiceProviderConfiguration'],
-    });
-    if (
-      registration.programFinancialServiceProviderConfigurationId ===
-      newFspConfig.id
-    ) {
-      const errors = `New FSP config id is the same as existing FSP config id for this Person Affected.`;
-      throw new HttpException({ errors }, HttpStatus.BAD_REQUEST);
-    }
-
-    // Get old registration to log
-    const oldViewRegistration =
-      await this.getPaginateRegistrationForReferenceId(
-        referenceId,
-        registration.programId,
-      );
-
-    // Update FSP
-    const updatedRegistration = await this.addFsp(referenceId, newFspConfig.id);
-    if (process.env.SYNC_WITH_THIRD_PARTIES) {
-      await this.sendContactInformationToIntersolve(updatedRegistration);
-    }
-
-    const newViewRegistration =
-      await this.getPaginateRegistrationForReferenceId(
-        referenceId,
-        registration.programId,
-      );
-    // Log change
-    await this.eventsService.log(oldViewRegistration, newViewRegistration, {
-      additionalLogAttributes: { reason: 'Financial service provider change' },
-    });
-
-    return newViewRegistration;
+    // const registration = await this.getRegistrationOrThrow({
+    //   referenceId,
+    //   relations: ['programFinancialServiceProviderConfiguration'],
+    // });
+    // if (
+    //   registration.programFinancialServiceProviderConfigurationId ===
+    //   newFspConfig.id
+    // ) {
+    //   const errors = `New FSP config id is the same as existing FSP config id for this Person Affected.`;
+    //   throw new HttpException({ errors }, HttpStatus.BAD_REQUEST);
+    // }
```

Comment:
Could this be "just removed"? Or is there another reason this is commented-out?

Reply:
Yes, make sense. I removed it in one of the other branches. So resolving this for now